### PR TITLE
Add back loss residual part

### PR DESCRIPTION
**Pull request by MARMOTatZJU**
* Add back loss residual part with builtin functinos in torch.nn.functional
* other minor fix in config file

verify sot alexnet
ao@val=0.732, ao@test=0.535

verify sot googlenet
ao@val=0.750, ao@test=0.607

### CODE REVIEW

**Review by lzx1413 — APPROVED** [review]
LGTM
